scripts/globals/pets/Excenmille.lua

```diff
@@ -0,0 +1,128 @@
+-------------------------------------------------
+--  TRUST: Excenmille
+--  Magic: Flash, Cure I - IV
+--  JA: Sentinel
+--  WS: Double Thrust, Leg Sweep, Penta Thrust   
+--  Source: http://bg-wiki.com/bg/Category:Trust
+-------------------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+
+function onMobSpawn(mob)
+    local weaponskill = 0
+	local cureCooldown = 18
+	local debuffCooldown = 10
+	local sentinelCooldown = 300
+	local master = mob:getMaster()
+	local excenmille = mob:getID()
+	mob:setLocalVar("cureTime",0)
+	mob:setLocalVar("debuffTime",0)
+	mob:setLocalVar("buffTime",0)
+	mob:setLocalVar("sentinelTime",0)
+	mob:setLocalVar("flashTime",0)
+
+	mob:addListener("TRUST_COMBAT_TICK", "TRUST_PROVOKE_TICK" .. excenmille, function(mob, player, target)
```

Comment:
do the event names need to have the id tacked on the end? you can't summon two excenmilles at the same time, right?  they also probably don't need their own name in it, calling it "PROVOKE_TICK" (without the id concatenated to the end) is good enough

Reply:
When I was testing, it required the ID because it caused trusts to take on weapon skills of the most recently spawned trust.  I tested again this morning after removing it and changing TRUST_COMBAT_TICK to COMBAT_TICK and it is no longer doing this.  Not sure what caused it.

Reply:
you might have been adding the listener to the player instead of the trust before, that would cause the issue you're describing